### PR TITLE
Tessellation API update

### PR DESCRIPTION
**Pull request by ktangsali**
# PhysicsNeMo Pull Request

## Description
Tessellation API changed in upstream. This fix updates sym implementation. 

## Checklist

- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/physicsnemo-sym/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/physicsnemo-sym/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/physicsnemo-sym/issues) is linked to this pull request.

### CODE REVIEW

File: physicsnemo/sym/geometry/tessellation.py [hunk 1]
```diff
@@ -118,47 +116,57 @@ def _sdf(triangles, airtight):
...
+                    import torch
+                    from physicsnemo.utils.sdf import signed_distance_field
+
+                    torch_vertices = torch.from_numpy(store_triangles)
+                    torch_indices = torch.arange(
+                        store_triangles.shape[0], dtype=torch.int32
+                    )
+                    torch_points = torch.from_numpy(points)
+
                     sdf_field, sdf_derivative = signed_distance_field(
```
[the remark this answers is not in the document]
Comment (ktangsali) [reply physicsnemo/sym/geometry/tessellation.py]: I agree, it is bit of a misnomer at the function return, but if we follow along the returned variable, the closest point is subtracted from the sampled point, giving the vector distance (and then normalized), giving the derivative

**Review by coreyjadams — APPROVED** [review]
